### PR TITLE
Semi-interactive script for GDPR user deletions

### PR DESCRIPTION
**Pull request by Brutus5000**

### CODE REVIEW

File: scripts/gdpr-delete.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                """, user_id)
+
+            if games == 0:
+                print("Player has no games! Starting complete account wipe!")
+
+                cursor.execute("DELETE FROM service_links WHERE user_id = %s and ownership = 0", user_id)
+                cursor.execute("DELETE FROM leaderboard_rating WHERE login_id = %s", user_id)
+                cursor.execute("DELETE FROM login WHERE id = %s", user_id)
+            else:
+                cursor.execute("UPDATE service_links sl SET user_id = null WHERE user_id = %s and ownership = 1",
```
Comment (Sheikah45) [inline scripts/gdpr-delete.py:124]: Doesn't this always need to be run? Even if they have no games. Since they still could have steam linked

Comment (Brutus5000) [reply scripts/gdpr-delete.py:124]: In the other case we delete all service links. If you didn't gain any rating, there is no reason to block the Steam/GOG account

File: scripts/gdpr-delete.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                               user_id)
+
+            while (input("Please confirm by typing CONFIRM: ") != 'CONFIRM'):
+                pass
+
+            # conn.commit()
```
Comment (Sheikah45) [inline scripts/gdpr-delete.py]: So we never commit the changes in the script XD

**Review by Sheikah45 — CHANGES_REQUESTED** [review]
Except for the commit being commented out looks good to me.

**Pull request merged**